products/run-run-and-sweep-github-files/artifacts.md

```diff
@@ -26,7 +26,7 @@ Click the link to the right of the artifact to copy the link
 ![](../../.gitbook/assets/im.png)
 
 {% hint style="info" %}
-Artifact links have an expiration of 24 hours.
+Artifact links have an expiration of 24 hours, new link is generated every 24 hours
```

Comment:
This looks good to me but I would try to emphasize that artifact data is never deleted unless asked by the user somehow.

Reply:
okay, added another line